src/netaddress.h

```diff
@@ -122,8 +122,16 @@ class CNetAddr
         unsigned int GetByte(int n) const;
         uint64_t GetHash() const;
         bool GetInAddr(struct in_addr* pipv4Addr) const;
-        std::vector<unsigned char> GetGroup() const;
+        std::vector<unsigned char> GetGroup(const std::vector<bool> &asmap) const;
```

Comment:
```suggestion
```

Reply:
Resolving this nit ended up nearly nuking this branch, thankfully I had a backup somewhere and restored it

Reply:
You may want to have a look at `git reflog` if you mess something up in your branch's history